index.js

```diff
@@ -1,131 +1,18 @@
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
-let max = process.argv[2];
-let min = 0;
-let guessCount = 0;
-
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
-async function start() {
-  console.log(`   _________________________________________________________
-  //=======================================================\\
-  \\..............WELCOME TO THE ONE AND ONLY..............//
-  //.................---------------------.................\\
-  \\.................||  GUESS WHAT!!!  ||.................//
-  //.................---------------------.................\\
-  \\=======================================================//
-   <><><><><><><><><><><><><><>*<><><><><><><><><><><><><><>\n\n`);
+start();
 
+async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  await numInt();
-  async function numInt() {
-    let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-    if (isNaN(parseInt(secretNumber)) || parseFloat(secretNumber) % 1 != 0) {    //Ensures user enters an integer
-      console.log("Please enter an integer...\n");
-      await numInt();
-    } else {
-      console.log('You entered: ' + secretNumber);
-      await playComp(secretNumber);
-    }
-  }
-}
-
-function guess(min, max) {
-  return Math.floor((parseInt(max)+min)/2); //Every guess is the middle of the remaining range of numbers
-}
-
-async function correctLetter (string, option1, option2) {
-  if (string.toUpperCase() === option1 || string.toUpperCase() === option2) {
-    return string.toUpperCase();
-  } else {
-    while (string.toUpperCase() !== option1 || string.toUpperCase() !== option2) {
-      string = await ask('Please enter a correct option...\n');
-      if (string.toUpperCase() === option1 || string.toUpperCase() === option2) {
-        return string.toUpperCase();
-      }
-      }
-  }
-}
-
-async function playComp(num) {
-  let numGuess = min + Math.floor(Math.random()*(max-min+1)); //First guess should be random to not give away our secret!
-  determine();
-  async function determine() {
-      let yesNo = await ask(`Is the number .... ${numGuess}? Y or N?\n`);
-      //if (yesNo.toUpperCase() === 'Y') {
```

Comment:
This looks nicer, and makes more sense to me than using the `correctLetter` function to check your input.  You could use an `else` block at the end of this logic block  to handle unexpected input.